exporter/datadogexporter/go.mod

```diff
@@ -1,4 +1,6 @@
-module github.com/DataDog/opentelemetry-collector-contrib/exporter/datadogexporter
+module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/datadogexporter
+
+replace gopkg.in/zorkian/go-datadog-api.v2 v2.29.0 => github.com/zorkian/go-datadog-api v2.29.1-0.20201007103024-437d51d487bf+incompatible
 
 go 1.15
```

Comment:
Not introduced here, but I believe the project has adopted 1.14?  Is this a required version?

Reply:
I think we changed core to 1.15 probably should update everywhere

Reply:
Should we address anything here then? It is not a required version and it only reflects the Go binary that created the go.mod file.

Reply:
I updated the `go.mod` file, it now shows `go 1.14`.